Doc/library/sqlite3.rst

```diff
@@ -1603,8 +1638,8 @@ registering custom adapter functions.
 
 .. _sqlite3-conform:
 
-Letting your object adapt itself
-""""""""""""""""""""""""""""""""
+Let your object adapt itself
+""""""""""""""""""""""""""""
```

Comment:
Even though these are subsections, iSTM they can be used as standalone howtos, since they are two alternative solutions.  In fact, I think it would be better to remove the "How to" from the previous section, and add it to the two subsections.

I also don't like this title too much.  What does "adapt itself" mean?  Why should I let my object do that?  Am I unknowingly preventing it to adapt?

```suggestion
How to write adaptable objects
""""""""""""""""""""""""""""""
```
I'm not sure the point of this howto can be summarized in a short title, but this at least informs me that there are adaptable objects and that by reading this section I'm going to learn how to create one (which might turn out to be useful).


Without the "How to" prefix and without the continuous form, these might be mistaken for imperative steps in the howto ("first let your object adapt itself, then register an adapter callable").

Reply:
For what its worth, I think that @ezio-melotti makes some great points and while it may or may not be optimal, I like his suggestion over what's there now, and don't mind having the discussion here, given the scope of this PR is specifically on improving these headings. But I'll defer to others on that call.

Reply:
SGTM; apply at will.